generator/internal/parser/openapi.go

```diff
@@ -242,10 +243,10 @@ func makePathTemplate(template string) []api.PathSegment {
 			}
 			if strings.HasPrefix(element, "{") && strings.HasSuffix(element, "}") {
 				element = element[1 : len(element)-1]
-				segments = append(segments, api.PathSegment{FieldPath: &element})
+				segments = append(segments, api.PathSegment{FieldPath: &api.FieldPath{Components: []*api.FieldPathComponent{{Identifier: element}}}})
```

Comment:
I think go experts would break this down in more lines for readability. Add some trailing commas and `go fmt` will do the expected thing.

```suggestion
				segments = append(segments, api.PathSegment{
				     FieldPath: &api.FieldPath{
				         Components: []*api.FieldPathComponent{
				             {Identifier: element},
				         },
				      },
				  })
```